gitlab-ce/inventory/group_vars/seed-hosts.yml

```diff
@@ -3,7 +3,9 @@ openshift_cluster_content:
 - object: projectrequest
   content:
   - name: gitlab-project
-    file: "{{ inventory_dir }}/../files/projects/projects.yml"
+    template: "https://raw.githubusercontent.com/redhat-cop/cluster-lifecycle/master/files/projectrequest/template.yml"
```

Comment:
@makentenza let's use a specific release here to avoid potential breakage in the future if/when this template is moved/changed.

Reply:
@oybed we haven't created cut releases on `cluster-lifecycle` so far. Can you create one?

Reply:
@makentenza hmm, interesting - I was positive we already had one. Anyway, I just cut a `v3.9.0` release:
https://github.com/redhat-cop/cluster-lifecycle/releases/tag/v3.9.0

Reply:
@oybed this is now done